maindataclasses.rb

```diff
@@ -0,0 +1,74 @@
+# task 1
+string1 = '235'
+sum = string1[0].to_i+string1[1].to_i+string1[2].to_i
```

Comment:
you don't know the length of the string
so please create a method which will take care of all string elements